index.html

```diff
@@ -19,101 +19,112 @@
         </header>
```

Comment:
- [ ] You didn't implement the nav menu, you can do that by adding a ```nav``` tag and the tree required links inside it ```Portfolio```, ```About```, and ```Contact```

![Screenshot_20220429_043803](https://user-images.githubusercontent.com/25227022/165885859-30cf0eb0-2b37-4bb9-b0be-f01728aaa5a1.png)